azurelinuxagent/common/osutil/alpine.py

```diff
@@ -34,7 +34,7 @@ def is_dhcp_enabled(self):
     def get_dhcp_pid(self):
         return self._get_dhcp_pid(["pidof", "dhcpcd"])
 
-    def restart_if(self, ifname): # pylint: disable=W0221
+    def restart_if(self, ifname, retries=None, wait=None):
```

Comment:
I'm surprised pylint is not complaining about unused variables now lol

Reply:
I was also confused with this, lol. My understanding is that pylint is smart enough to realize that these parameters are used in the base classes' method, that this method is overriding.